esx_service/vmdk_ops_test.py

```diff
@@ -42,14 +42,23 @@
 import random
 import convert
 from error_code import ErrorCode
+from vmdkops_admin_sanity_test import ADMIN_CLI
+import glob
 
 # Max volumes count we can attach to a singe VM.
 MAX_VOL_COUNT_FOR_ATTACH = 60
 
+# Admin CLI to control config DB init
+ADMIN_INIT_LOCAL_AUTH_DB = ADMIN_CLI + " config init --local"
+ADMIN_RM_LOCAL_AUTH_DB = ADMIN_CLI + " config rm --local --confirm"
+
+# backups to cleanup
+CONFIG_DB_BAK_GLOB = "/etc/vmware/vmdkops/auth-db.bak_*"
+
 # Seed for test configurations.
 config = {
     # If True, test 60+ attaches (no detach) until if fails.
-    "run_max_attach": True
+    "run_max_attach": False
```

Comment:
@shuklanirdesh82  and I discussed it offline. It take quit a long time and has low code coverage, so we want to accelerate test passes. Nirdesh has it on the list of things to bring back once we get control over test timing